src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends MainApp.Machine {

```

Comment:
Once you externalize `Machine` as a separate public class, this prefix won't be necessary.